src/main/java/uk/gov/hmcts/auth/provider/service/api/config/AppProperties.java

```diff
@@ -12,7 +12,7 @@
 
     private int ttlInSeconds = (int) HOURS.toSeconds(4);
 
-    private Map<String, String> microserviceKeys;
+    private Map<String, String> serviceKeys;
```

Comment:
This change will break all teams docker images, I don't think we should do it like this, haven't we already tested that the secrets have been migrated correctly?

Reply:
You might be right that this may be an unnecessary step. Just wanted to be more careful, but we can always revert to the previous state.

Reply:
I've applied your initial suggestion and this PR simply replaces the old settings with the new ones, so that the applications reads the same environment variables now.